src/main/scala/shade/memcached/Memcached.scala

```diff
@@ -15,31 +18,31 @@ trait Memcached extends java.io.Closeable {
    *
    * @return either true, in case the value was set, or false otherwise
    */
-  def add[T](key: String, value: T, exp: Duration)(implicit codec: Codec[T]): Future[Boolean]
+  def addF[T](key: String, value: T, exp: Duration)(implicit codec: Codec[T]): CancelableFuture[Boolean]
 
   def awaitAdd[T](key: String, value: T, exp: Duration)(implicit codec: Codec[T]): Boolean =
-    Await.result(add(key, value, exp), Duration.Inf)
+    Await.result(addF(key, value, exp), Duration.Inf)
 
   /**
    * Sets a (key, value) in the cache store.
    *
    * The expiry time can be Duration.Inf (infinite duration).
    */
-  def set[T](key: String, value: T, exp: Duration)(implicit codec: Codec[T]): Future[Unit]
+  def setF[T](key: String, value: T, exp: Duration)(implicit codec: Codec[T]): CancelableFuture[Unit]
```

Comment:
Dude, I just looked at the Monix implementation and the `F` suffix convention is used for something else (it's for operations that keep the original context, like `foldLeftF` which still returns an `Observable`).  We'll use the `L` suffix for `Task`, but for `Future` return types we haven't used any convention and we shouldn't do it here.

Sorry for misleading you :-P